examples/localclient_with_etcd/main.go

```diff
@@ -0,0 +1,133 @@
+//  Copyright (c) 2019 Cisco and/or its affiliates.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at:
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/ligato/cn-infra/agent"
+
+	"github.com/ligato/cn-infra/datasync"
+	"github.com/ligato/cn-infra/datasync/kvdbsync"
+	"github.com/ligato/cn-infra/datasync/kvdbsync/local"
+	"github.com/ligato/cn-infra/datasync/resync"
+	"github.com/ligato/cn-infra/db/keyval/etcd"
+	"github.com/ligato/cn-infra/health/statuscheck"
+	"github.com/ligato/vpp-agent/api/models/vpp/interfaces"
+	//legacyclient "github.com/ligato/vpp-agent/clientv2/linux/localclient"
+	"github.com/ligato/vpp-agent/client"
+	kvs "github.com/ligato/vpp-agent/plugins/kvscheduler"
+	"github.com/ligato/vpp-agent/plugins/orchestrator"
+	vpp_ifplugin "github.com/ligato/vpp-agent/plugins/vpp/ifplugin"
+)
+
+func main() {
+	ep := &ExamplePlugin{
+		KVScheduler:  &kvs.DefaultPlugin,
+		VPPIfPlugin:  &vpp_ifplugin.DefaultPlugin,
+		Orchestrator: &orchestrator.DefaultPlugin,
+		ETCDDataSync: kvdbsync.NewPlugin(kvdbsync.UseKV(&etcd.DefaultPlugin)),
+	}
+
+	writers := datasync.KVProtoWriters{
+		ep.ETCDDataSync,
+	}
+	statuscheck.DefaultPlugin.Transport = writers
+
+	watchers := datasync.KVProtoWatchers{
+		local.DefaultRegistry,
+		ep.ETCDDataSync,
+	}
+	orchestrator.DefaultPlugin.Watcher = watchers
+
+	a := agent.NewAgent(
+		agent.AllPlugins(ep),
+	)
+	if err := a.Run(); err != nil {
+		log.Fatal(err)
+	}
+}
+
+// ExamplePlugin is the main plugin which
+// handles resync and changes in this example.
+type ExamplePlugin struct {
+	KVScheduler  *kvs.Scheduler
```

Comment:
Do we need to have `KVScheduler` specifically listed, even if it's not used by the `ExamplePlugin`? The orchestrator plugin contains default KVScheduler instance already and is listed already by the `ExamplePlugin`.

Reply:
another mistake (the example was copy&pasted from another one which actually does something with the scheduler) - fixed